Dockerfile

```diff
@@ -0,0 +1,30 @@
+# First stage: build environment
+FROM python:3.11-bullseye AS build-env
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the requirements file
+COPY requirements.txt .
+
+# Install the dependencies
+RUN pip install --no-cache-dir --upgrade pip && \
+    pip install --no-cache-dir -r requirements.txt
+
+# Second stage: runtime environment
+FROM python:3.11-slim
+
+# Set the working directory to /app
+WORKDIR /app
+
+RUN mkdir -p /root/.config/
+
+# Copy the script and libraries
+COPY ankerctl.py /app/ankerctl.py
+COPY libflagship /app/libflagship
+COPY cli /app/cli
```

Comment:
Would this work?

`COPY ankerctl.py libflagship/ cli/ /app/`?

Reply:
Yes, tested and changed